arches_lingo/views/api/concepts.py

```diff
@@ -91,3 +94,139 @@ def default_sensitivity():
         if elastic_prefix_length >= 5:
             return 0
         return int(5 - elastic_prefix_length)
+
+
+@method_decorator(
+    group_required("RDM Administrator", raise_exception=True), name="dispatch"
+)
+class ConceptResourceView(ConceptTreeView):
+    def get(self, request):
+        scheme = request.GET.get("scheme", None)
+        exclude = request.GET.get("exclude", None)
+        term = request.GET.get("term", None)
+        page_number = request.GET.get("page", 1)
+        items_per_page = request.GET.get("items", 25)
+        concepts = request.GET.get("concepts", None)
+        concept_ids = concepts.split(",") if concepts else None
+        Concept = SemanticResource.as_model("concept")
+
+        if not concept_ids:
+            # if scheme:
+            #     if exclude == "true":
+            #         concept_query = Concept.exclude(
+            #             part_of_scheme__0__0__resourceId=scheme
+            #         )
+            #     else:
+            #         concept_query = Concept.filter(
+            #             part_of_scheme__0__0__resourceId=scheme
+            #         )
+            # else:
+            concept_query = Concept.all()
+
+            if term:
+                filtering_concept_ids = VwLabelValue.objects.filter(
```

Comment:
I have a branch that gets rid of VwLabelValu in order to stick with `Concept`. I'll ask you to review it when it's ready soon, just a heads up